app/assets/stylesheets/application.scss

```diff
@@ -35,6 +35,9 @@ $hmrc-assets-path: "/report-quarterly/income-and-expenses/view/assets/lib/hmrc-f
 
 }
 
+.bta-nav-no-border{
```

Comment:
Should fix: if there is a need to remove border you will need to override the border property.